lib/mongoid/interceptable.rb

```diff
@@ -171,8 +175,8 @@ def pending_callbacks
     # @api private
     def run_pending_callbacks
       pending_callbacks.each do |cb|
-        if cb == :apply_defaults
-          self.apply_defaults
+        if [:apply_defaults, :apply_post_processed_defaults].include? cb
```

Comment:
I believe we use parenthesis for method calls.

Reply:
done